Practical-Forms/Petition.txt

```diff
@@ -4,11 +4,11 @@
 [color=#aaaaaa]▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬[/color]
 ►──────
   [head=2]Petition Demands:[/head]
-The Signees of this Petition have the following demands:
+The Signers of this Petition have the following demands:
 [color=#002AAF]Sample Demands[/color]
 
 ►──────
-  [head=2]Petition Signees:[/head]
+  [head=2]Petition Signers:[/head]
```

Comment:
Signee is a [real word](https://www.britannica.com/dictionary/signee) that is used correctly here.